stubs/protobuf/google/protobuf/text_format.pyi

```diff
@@ -0,0 +1,204 @@
+import sys
+from _typeshed import SupportsWrite
+from typing import Any, Callable, Iterable, Optional, Text, Tuple, TypeVar, Union
+
+from .descriptor import FieldDescriptor
+from .descriptor_pool import DescriptorPool
+from .message import Message
+
+if sys.version_info < (3, 0):
+    long = int
+
+_M = TypeVar("_M", bound=Message)  # message type (of self)
+
+class Error(Exception): ...
+
+class ParseError(Error):
+    def __init__(self, message: Optional[str] = ..., line: Optional[int] = ..., column: Optional[int] = ...) -> None: ...
+    def GetLine(self) -> Optional[int]: ...
+    def GetColumn(self) -> Optional[int]: ...
+
+class TextWriter:
+    def __init__(self, as_utf8: bool) -> None: ...
+    if sys.version_info < (3, 0):
+        def write(self, val: Union[Text, bytes]) -> int: ...
+    else:
+        def write(self, val: str) -> int: ...
+    def getvalue(self) -> str: ...
+    def close(self) -> None: ...
+
+_MessageFormatter = Callable[[Message, int, bool], Optional[Text]]
+
+def MessageToString(
+    message: Message,
+    as_utf8: bool = ...,
+    as_one_line: bool = ...,
+    use_short_repeated_primitives: bool = ...,
+    pointy_brackets: bool = ...,
+    use_index_order: bool = ...,
+    float_format: Optional[str] = ...,
+    double_format: Optional[str] = ...,
+    use_field_number: bool = ...,
+    descriptor_pool: Optional[DescriptorPool] = ...,
+    indent: int = ...,
+    message_formatter: _MessageFormatter = ...,
+    print_unknown_fields: bool = ...,
+    force_colon: bool = ...,
+) -> str: ...
+def MessageToBytes(message: Message, **kwargs: Any) -> bytes: ...
```

Comment:
The kwargs just get passed to MessageToString, so you can copy its kwargs instead of using `**kwargs: Any`.